local_php.ini

```diff
@@ -0,0 +1,1904 @@
+[PHP]
+
+;;;;;;;;;;;;;;;;;;;
+; About php.ini   ;
+;;;;;;;;;;;;;;;;;;;
+
+; Disable aggressive caching
+opcache.enable = 0
```

Comment:
Key idea is this parameter to disable caching. That way, edits to .php files take effect immediately.